Base/usr/share/man/man1/FontEditor.md

```diff
@@ -27,8 +27,8 @@ Presently, you can resize the entire FontEditor form to your liking. You can tur
 
 ### Contents of the Menu Bar
```

Comment:
I'm not sure it makes sense to even list these here. Not sure what value it provides.

Reply:
What would you rather suggest to improve on this section of the document?

Reply:
I would suggest not having this section at all. Listing the contents of the menu just seems like a maintenance burden with no tangible benefit.

Reply:
This was done.